attribute-event-requests/INDI-attribute.md

```diff
@@ -63,185 +56,218 @@ Related proposals include
-which might mean there are children that have not yet been added to the data.
+### Value
 
-### Used
 
-- Part of the [GEDCOM X specification](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) as a distinct stucture with URI `http://familysearch.org/v1/CoupleNeverHadChildren`
+### Absence
 
 
-## Clan
+### Used
+- Part of the GEDCOM X specification as a distinct stucture with URI http://familysearch.org/v1/CoupleNeverHadChildren
 
+--------------------------
+## Clan
 ### Description
-
 *proposed description missing*
-
-*In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) as* "A fact of a person's clan."
+*In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) as*  "A fact of a person's clan."
 
 ### Value
-
 Found in the following historical records:
 
 - (records not yet identified)
 
 ### Absence
-
 The most closely related structures are:
 
-- 
-
 Related proposals include
 
 - Tribe: *difference from Clan not yet articulated*
 - Ethnicity: *difference from Clan not yet articulated*
-- Heimat: a place, often as a disambiguation of family name or lineage. Clan suggests group identity, not geography
-- Affiliation: generally by choice or appointment, while clan generally indicates something assigned to a person at birth
+- Heimat: *a place, often as a disambiguation of family name or lineage. Clan suggests group identity, not geography*
+- Affiliation: *generally by choice or appointment, while clan generally indicates something assigned to a person at birth*
 
 ### Used
 
-- Part of the [GEDCOM X specification](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Clan`
-
-- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) via GEDCOM X.
 
+- [GEDCOM X specification](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Clan`
+- [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) via GEDCOM X.
 
+--------------------------
 ## Died Before Eight
+### Description
+The assertion that a person died before the end of their eighth year of life can be made using the `DEAT` structure with an AGE substructure with payload "< 8y".
 
-The assertion that a person died before the end of their eighth year of life can be made using the `DEAT` structure with an `AGE` substructure with payload "`< 8y`".
+### Value
 
-### Used
 
-- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) with URI `http://familysearch.org/v1/DiedBeforeEight`
+### Absence
 
 
-## Ethnicity
+### Used
+- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) with URI http://familysearch.org/v1/DiedBeforeEight
```

Comment:
```suggestion
- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) with URI `http://familysearch.org/v1/DiedBeforeEight`
```
I think the general rule for whether to wrap URIs in backticks is to use backticks around URNs and no backticks around URLs.  I.e., whether they can be clicked to resolve to something or not.